Cooper/ORM/CRUD.cs

```diff
@@ -59,118 +59,23 @@ public long Create(string table, string idColumn, EntityORM entity, bool returni
-                {
-                    entity = new EntityORM();
-                    foreach (string attribute in attributes)
-                    {
-                        object value = reader[attribute];
-                        entity.attributeValue.Add(attribute, value);
-                    }
-                }
-
-                reader.Close();
-            }
-            catch (DbException ex)
-            {
-                logger.Info("Exception.Message: {0}", ex.Message);
-            }
-            finally
-            {
-                dbConnect.CloseConnection();
-            }
-
-
-            return entity;
-        }
-
-        public List<string> ReadFieldValues(string field, string table)
-        {
-            List<string> fieldValues = new List<string>();
-            try
-            {
-                string sqlExpression = $"SELECT {field} FROM {table}";
-
-                dbConnect.OpenConnection();
-                OracleCommand command = new OracleCommand(sqlExpression, dbConnect.GetConnection());
-
-                OracleDataReader reader = command.ExecuteReader();
-
-                while (reader.Read())
-                {
-                    fieldValues.Add(reader[field].ToString());
-                }
-
-                reader.Close();
-            }
-            catch (DbException ex)
-            {
-                logger.Info("Exception.Message: {0}", ex.Message);
-            }
-            finally
-            {
-                dbConnect.CloseConnection();
-            }
-
-            return fieldValues;
-        }
-
-        public IEnumerable<EntityORM> ReadBellow(object attribute_value, string attribute_name, HashSet<string> attributes, string table) {
+        public IEnumerable<EntityORM> Read(string table, HashSet<string> attributes, DbTools.WhereRequest[] whereRequests = null)
+        {   
             List<EntityORM> entities = new List<EntityORM>();
-
             try
             {
-                string sqlExpression = $"SELECT * FROM {table} WHERE {attribute_name} <= {attribute_value}";
-
-                dbConnect.OpenConnection();
-                OracleCommand command = new OracleCommand(sqlExpression, dbConnect.GetConnection());
-
-                OracleDataReader reader = command.ExecuteReader();
-                while (reader.Read())
-                {
-                    EntityORM entity = new EntityORM(); 
-                    foreach (string attribute in attributes)
+                string where = "";
+                if (whereRequests != null) {
+                    where = " WHERE";
+                    foreach (DbTools.WhereRequest request in whereRequests)
                     {
-                        object value = reader[attribute];
-                        entity.attributeValue.Add(attribute, value);
+                        where += String.Format(" {0} {1} {2},", request.variable_name, DbTools.GetOperatorString(request.request_operator), request.variable_value);
```

Comment:
if you have many filters, probably "AND" should be used. Actually here will be much complicated logic to allow ORs and ANDs.